plutus-pab/src/Plutus/PAB/Types.hs

```diff
@@ -95,24 +100,35 @@ instance Pretty PABError where
         RemoteWalletWithMockNodeError   -> "The remote wallet can't be used with the mock node."
         TxSenderNotAvailable         -> "Cannot send a transaction when connected to the real node."
 
-data DbConfig =
-    DbConfig
-        { dbConfigFile     :: Text
-        -- ^ The path to the sqlite database file. May be absolute or relative.
-        , dbConfigPoolSize :: Int
-        -- ^ Max number of concurrent sqlite database connections.
-        }
+data DBConnection = PostgresPool (Pool Postgres.Connection)
+                  | SqlitePool (Pool Sqlite.Connection)
+
+data DbConfig = SqliteDB Sqlite.DbConfig
+              | PostgresDB Postgres.DbConfig
     deriving (Show, Eq, Generic)
-    deriving anyclass (ToJSON, FromJSON)
+
+instance FromJSON DbConfig where
+    parseJSON (Object obj) = do
+        ci <- obj .:? "sqliteDB"
+        bf <- obj .:? "postgresDB"
+        case (ci, bf) of
+            (Just a, Nothing)  -> pure $ SqliteDB a
+            (Nothing, Just a)  -> pure $ PostgresDB a
+            (Nothing, Nothing) -> error $ unwords
+                                  [ "No configuration available, expecting"
+                                  , "sqliteDB or postgresDB"
```

Comment:
As people migrate, they will need to update their config file. Would be nice to explicitly state in the message that `DbConfig` was removed and a user need to use `sqliteDB` or `postgresDB` fields.

Also, it's not `postgres`, it's `postgresql`. Would be good to do naming properly.

Reply:
I agree. Just commited the adjustments made the adjustments.